### PR TITLE
Simplify cgo install instructions

### PR DESCRIPTION
**Pull request by kortschak**
Drop reference to $GOPATH.

### CODE REVIEW

**Comment by jonlawlor** [comment]
LGTM

**Comment by kortschak** [comment]
PTAL

File: README.md [hunk 1]
```diff
@@ -17,10 +17,9 @@ If you want to use OpenBLAS, install it in any directory:
   make
 ```
```
Comment (btracey) [inline README.md]: While we're here, could we add a comment that CGO_LDFLAGS can be set as an environment variable? I would guess we'll eventually have people who won't understand that from the current documentation. A high-level comment along the lines of "The cgo package provides bindings to c-backed BLAS packages. blas/cgo needs the CGO_LDFLAGS environment variable to point to the blas installation. More information can be found in the cgo documentation [link to http://golang.org/cmd/cgo/] "

Comment (kortschak) [inline README.md]: Is that our job?

Comment (kortschak) [inline README.md]: Done.

Comment (btracey) [inline README.md]: If you want you can leave out the middle sentence and just have "The blas/cgo package provides bindings to external BLAS libraries. More information about using cgo can be found at [link to http://golang.org/cmd/cgo/] "

Two sentences aren't a lot of cost, and it'll point people in the right direction if they're confused.

Comment (kortschak) [inline README.md]: PTAL

**Comment by btracey** [comment]
Thanks. LGTM.

**Pull request merged**